source/plugins/go/src/extension/extension_windows.go

```diff
@@ -0,0 +1,35 @@
+//go:build windows
+
+package extension
+
+import (
+
+	winio "github.com/Microsoft/go-winio"
+)
+
+func getExtensionConfigResponse(jsonBytes []byte) (string, error) {
+	pipePath := `\\.\\pipe\\CAgentStream_CloudAgentInfo_AzureMonitorAgent`
+	config_namedpipe, err := winio.DialPipe(pipePath, nil)
+	if err != nil {
+		logger.Printf("Windows AMA: error opening pipe: %v", err)

```

Comment:
include pipePath in the error message so its easy to troubleshoot from the logs.